frontend/src/components/Navigation.vue

```diff
@@ -25,7 +28,16 @@ export default {
       steps: 1
     };
   },
-
+  methods: {
+    getStepClass(currentStep, index) { 
+      return {'step-linkable' : currentStep > index};
+    },
+    gotoStep(numOfSteps) { // numOfSteps is number of steps to go back
```

Comment:
back is better in my opinion

Reply:
it's basically 'go to step'  since it's getting a value how many steps to go back/fwd.